NEWS.md

```
@@ -114,6 +114,7 @@ In the deprecated functions:
 
 ## pgRouting 3.7
 
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)